landoscript/src/landoscript/actions/merge_day.py

```diff
@@ -80,18 +82,20 @@ async def run(github_client: GithubClient, public_artifact_dir: str, merge_info:
         bump_version = await get_version(github_client, version_file, from_branch)
         bump_branch = from_branch
         log.info(f"from_branch is present, got bump_version from it: {bump_version}")
+        bump_ref = await github_client.get_branch_head_oid(from_branch)
 
         # base tagging _only_ happens when we have a `from_branch` -- these are
         # scenarios where we're uplifting one branch to another, and beginning a new
         # version number on the `to_branch`, which we declare with the `BASE` tag.
         if base_tag:
             base_tag_fmted = base_tag.format(major_version=bump_version.major_number)
             log.info(f"Adding base_tag: {base_tag_fmted}")
-            actions.extend(tag.run([base_tag_fmted]))
+            actions.extend(await tag.run(session, tag.GitTagInfo(revision=bump_ref, tags=[base_tag_fmted])))
     else:
         if merge_old_head:
             raise TaskVerificationError("'from_branch' is required when merge_old_head is True or version_files are present")
 
+        bump_ref = from_branch
```

Comment:
nit: dead store (we've just asserted that !merge_old_head so bump_ref won't get used)